packages/cache/src/cache.ts

```diff
@@ -100,6 +100,7 @@ export async function restoreCache(
       options
     )
 
+    await listTar(archivePath, compressionMethod)
```

Comment:
Listing out the contents of the tar file might not be ideal in all scenarios. If users are concerned more about cache speed than log verbosity, we should consider making this optional.

Reply:
Thanks for your review. I've added it under `if (core.isDebug())` conditinal.

Reply:
I'd like to note that `@actions/glob` also has its own debug output, but it's not the same as `listTar`:

![image](https://user-images.githubusercontent.com/2205188/101363730-caa3d200-38b2-11eb-8eb4-42e26845681f.png)